django/contrib/messages/apps.py

```diff
@@ -1,7 +1,16 @@
 from django.apps import AppConfig
+from django.contrib.messages.storage import base
+from django.dispatch import receiver
+from django.test.signals import setting_changed
 from django.utils.translation import gettext_lazy as _
 
 
 class MessagesConfig(AppConfig):
     name = 'django.contrib.messages'
     verbose_name = _("Messages")
+
+
+@receiver(setting_changed)
+def reset_hashers(setting, value, **kwargs):
+    if setting == 'MESSAGE_TAGS':
+        base.LEVEL_TAGS = value
```

Comment:
wrong function name (`reset_hashers`) I think? :)
Also does the introduction of this obviate the need for the `override_settings_tags` class in `messages_tests.tests`?

Reply:
> wrong function name (reset_hashers) I think? :)

Yeah, Thanks for pointing. It was leftover from the copy :)

> Also does the introduction of this obviate the need for the override_settings_tags class in messages_tests.base?

No sure!

Reply:
Yes, `override_settings_tags()` should no longer be necessary. Please change it to the `override_settings()`.

Reply:
Done!